ansible/inventories/production/group_vars/inventory-next/vars.yml

```diff
@@ -40,7 +40,7 @@ inventory_ckan_plugins_additional: [s3filestore saml2auth]
 # saml2 authentication
 inventory_ckan_saml2_enabled: true
 ckan_production_ini_template: etc/ckan/production.ini
-inventory_ckan_who_ini_path: etc_ckan_who.default.ini.j2
+inventory_ckan_who_ini_path: etc_ckan_who.saml2.ini.j2
```

Comment:
`etc_ckan_who.saml2.ini.j2` is who.ini for ckanext-saml2 for max.log.
`inventory-next/etc_ckan_who.saml2.ini.j2` is who.ini for ckanext-saml2auth for login.gov

Let me make another PR.

Reply:
https://github.com/GSA/datagov-deploy/pull/2765